examples/api/test/widgets/raw_menu_anchor/raw_menu_anchor.1_test.dart

```diff
@@ -0,0 +1,150 @@
+    expect(find.text('Cut'), findsOneWidget);
+    expect(find.text('Copy'), findsOneWidget);
+    expect(find.text('Paste'), findsOneWidget);
+    expect(find.text('Format'), findsOneWidget);
+    expect(find.text('Bold'), findsNothing);
+    expect(find.text('Italic'), findsNothing);
+    expect(find.text('Underline'), findsNothing);
+
+    await tester.sendKeyEvent(LogicalKeyboardKey.arrowDown);
+    await tester.sendKeyEvent(LogicalKeyboardKey.arrowDown);
+    await tester.sendKeyEvent(LogicalKeyboardKey.arrowDown);
+    await tester.sendKeyEvent(LogicalKeyboardKey.arrowDown);
+    await tester.sendKeyEvent(LogicalKeyboardKey.arrowDown);
+    await tester.pump();
+    await tester.sendKeyEvent(LogicalKeyboardKey.arrowRight);
+    await tester.pump();
+
+    expect(primaryFocus!.debugLabel, equals('MenuItemButton(Text("Bold"))'));
+    expect(tester.getSemantics(find.text('Format')).hasFlag(SemanticsFlag.isExpanded), isTrue);
+    expect(find.text('Undo'), findsOneWidget);
+    expect(find.text('Redo'), findsOneWidget);
+    expect(find.text('Cut'), findsOneWidget);
+    expect(find.text('Copy'), findsOneWidget);
+    expect(find.text('Paste'), findsOneWidget);
+    expect(find.text('Format'), findsOneWidget);
+    expect(find.text('Bold'), findsOneWidget);
+    expect(find.text('Italic'), findsOneWidget);
+    expect(find.text('Underline'), findsOneWidget);
+
+    await tester.sendKeyEvent(LogicalKeyboardKey.arrowLeft);
+    await tester.pump();
+
+    expect(primaryFocus!.debugLabel, equals('MenuItemButton(Text("Format"))'));
+    expect(tester.getSemantics(find.text('Format')).hasFlag(SemanticsFlag.isExpanded), isFalse);
+    expect(find.text('Bold'), findsNothing);
+
+    await tester.sendKeyEvent(LogicalKeyboardKey.arrowDown);
+    await tester.pump();
+
+    expect(primaryFocus!.debugLabel, equals('MenuItemButton(Text("Undo"))'));
+
+    await tester.sendKeyEvent(LogicalKeyboardKey.enter);
+    await tester.pump();
+    await tester.pump();
+
+    expect(find.text('Selected: Undo'), findsOneWidget);
+  });
+
+  testWidgets('Platform Brightness does not affect menu appearance', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const MediaQuery(
+        data: MediaQueryData(platformBrightness: Brightness.dark),
+        child: example.ContextMenuApp(),
+      ),
+    );
+
+    await tester.tapAt(const Offset(100, 200), buttons: kSecondaryButton);
+    await tester.pump();
+
+    expect(
+      findMenuPanelDescendent<Container>(tester).decoration,
+      RawMenuAnchor.defaultLightOverlayDecoration,
+    );
+  });
+
+  testWidgets('Browser context menu is disabled', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.ContextMenuApp());
+
+    await tester.tapAt(const Offset(100, 200), buttons: kSecondaryButton);
+    await tester.pump();
+
+    if (kIsWeb) {
+      assert(BrowserContextMenu.enabled, isFalse);
+    }
+  },
+  skip: !kIsWeb // Browser context menu is only enabled on the web
+  );
```

Comment:
```suggestion
  }, skip: !kIsWeb); // [intended] Browser context menu is only enabled on the web
```

`[intended]` text is required to please `Linux analyze` check.